lib/bosh/index.js

```diff
@@ -6,28 +6,13 @@ const EvaluationContext = require('./EvaluationContext');
 const Networks = require('./manifest/Networks');
 const Network = require('./manifest/Network');
 const Header = require('./manifest/Header');
-const config = require('../config');
-const CONST = require('../constants');
 
 exports.director = new BoshDirectorClient();
 exports.NetworkSegmentIndex = NetworkSegmentIndex;
 exports.EvaluationContext = EvaluationContext;
 exports.BoshDirectorClient = BoshDirectorClient;
```

Comment:
return back an instance of BoshDirectorClient. BoshDirectorClient must be made as singleton and everyother place must just use the same instance and must not create their own instance. Creating new instances leads to creating of those many # of caches / calls to boshs unnecessarily to build cache.
```
exports.boshDirectorClient = new BoshDirectorClient();
````
[Please note lowercase of 'b' in the above. Lowercase indicates an instance in all other places in code. Update references accordingly all other places.]